### ISSUE TITLE
LaraAdmin for L5.4

### ISSUE THREAD

**Issue by xJoeyv** [dwijitsolutions/laraadmin#210]
I only see that is this currently working on Laravel 5.2.. That is 2 versions behind on the current Laravel version. Will this still be updated or what? Since this looks as a Amazing Admin solution

**Comment by chrispecoraro** [comment]
There have been a few different attempts as this, including mine: https://github.com/dwijitsolutions/laraadmin/pull/153 -- take a look at the pull requests.

**Comment by jocarullo** [comment]
yes please work on this especially its been almost a year since the fix for laraadmin to work for laravel 5.3, we appreciate your work.
